arm-apimanagement/2016-10-10/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1270 @@
+          },
+          "description": "Custom hostname configuration of the API Management service."
+        },
+        "staticIPs": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Static IP addresses of the API Management service virtual machines. Available only for Standard and Premium SKU."
+        },
+        "vpnconfiguration": {
+          "$ref": "#/definitions/VirtualNetworkConfiguration",
+          "description": "Virtual network configuration of the API Management service."          
+        },
+        "additionalLocations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AdditionalRegion"
+          },
+          "description": "Additional datacenter locations of the API Management service."
+        },
+        "customProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Custom properties of the API Management service, like disabling TLS 1.0."
+        },
+        "vpnType": {
+          "type": "string",
+          "description": "The type of VPN in which API Managemet service needs to be configured in. None (Default Value) means the API Management service is not part of any Virtual Network, External means the API Management deployment is set up inside a Virtual Network having an Internet Facing Endpoint, and Internal means that API Management deployment is setup inside a Virtual Network having an Intranet Facing Endpoint only.",
+          "default": "None",
+          "enum": [
+            "None",
+            "External",
+            "Internal"
+          ],
+          "x-ms-enum": {
+            "name": "VirtualNetworkType",
+            "modelAsString": false
+          }
+        }        
+      },
+      "required": [
+        "publisherEmail",
+        "publisherName"
+      ],
+      "description": "Properties of an API Management service resource description."
+    },
+    "ApiManagementServiceSkuProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Sku.",
+          "externalDocs": {
+            "url": "https://azure.microsoft.com/en-us/pricing/details/api-management/"
+          },
+          "enum": [
+            "Developer",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuType",
+            "modelAsString": false
+          }
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Capacity of the SKU (number of deployed units of the SKU). The default value is 1.",
+          "default": 1
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "API Management service resource SKU properties."
+    },
+    "ApiManagementServiceResource": {
```

Comment:
Is `ApiManagementServiceResource` an ARM resource? if yes any reason not to define `Resource` ("x-ms-azure-resource") model and allOf from it?